### PR TITLE
Update package maintainers to include all past and present core team members

### PR DESCRIPTION
**Pull request by neersighted**
@python-poetry/core, please check the attribution in `pyproject.toml` and make sure it looks good/give it your ✅.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -3,7 +3,15 @@ name = "poetry"
 version = "1.2.0-beta.2.dev0"
 description = "Python dependency management and packaging made easy."
 authors = [
-    "Sébastien Eustace <sebastien@eustace.io>"
+    "Sébastien Eustace <sebastien@eustace.io>",
+    "Arun Babu Neelicattu <arun.neelicattu@gmail.com>",
```
Comment (Secrus) [inline pyproject.toml:7]: Wouldn't it fit better to leave sdispater as author and add rest of the team as `maintainers`?

**Review by Secrus — APPROVED** [review]

File: pyproject.toml (hunk 1 above)
Comment (neersighted) [reply pyproject.toml:7]: You know, I forgot that field existed in the Python metadata...

**Review by abn — COMMENTED** [review]
Approved. Unless @radoering or @finswimmer prefers to update details for their entries.

**Review by abn — DISMISSED** [review]

**Review by branchvincent — DISMISSED** [review]

File: pyproject.toml [hunk 2]
```diff
@@ -3,7 +3,17 @@ name = "poetry"
...
-    "Sébastien Eustace <sebastien@eustace.io>"
+    "Sébastien Eustace <sebastien@eustace.io>",
+]
+maintainers = [
+    "Arun Babu Neelicattu <arun.neelicattu@gmail.com>",
+    "Bjorn Neergaard <bjorn@neersighted.com>",
+    "Branch Vincent <branchevincent@gmail.com>",
+    "Bryce Drennan <github@accounts.brycedrennan.com>",
+    "Daniel Eades <danieleades@hotmail.com>",
+    "Randy Döring <30527984+radoering@users.noreply.github.com>",
```
Comment (radoering) [inline pyproject.toml:14]:
```suggestion
    "Randy Döring <radoering.poetry@gmail.com>",
```
I am already curious what kind of mails I will receive. 😉

**Review by abn — APPROVED** [review]

**Pull request merged**